node-test/src/test/scala/org/bitcoins/node/UnsyncedNeutrinoNodeTest.scala

```diff
@@ -0,0 +1,110 @@
+package org.bitcoins.node
+
+import org.bitcoins.asyncutil.AsyncUtil
+import org.bitcoins.crypto.DoubleSha256DigestBE
+import org.bitcoins.rpc.client.common.BitcoindRpcClient
+import org.bitcoins.server.BitcoinSAppConfig
+import org.bitcoins.testkit.BitcoinSTestAppConfig
+import org.bitcoins.testkit.node.fixture.NeutrinoNodeConnectedWithBitcoinds
+import org.bitcoins.testkit.node.{NodeTestUtil, NodeTestWithCachedBitcoindPair}
+import org.scalatest.{Assertion, FutureOutcome, Outcome}
+
+import scala.concurrent.Future
+import scala.concurrent.duration.DurationInt
+
+class UnsyncedNeutrinoNodeTest extends NodeTestWithCachedBitcoindPair {
+
+  override protected def getFreshConfig: BitcoinSAppConfig =
+    BitcoinSTestAppConfig.getMultiPeerNeutrinoWithEmbeddedDbTestConfig(pgUrl)
+
+  override type FixtureParam = NeutrinoNodeConnectedWithBitcoinds
+
+  override def withFixture(test: OneArgAsyncTest): FutureOutcome = {
+    val outcomeF: Future[Outcome] = for {
+      bitcoinds <- clientsF
+      outcome = withUnsyncedNeutrinoNodeConnectedToBitcoinds(
+        test,
+        bitcoinds.toVector)(system, getFreshConfig)
+      f <- outcome.toFuture
+    } yield f
+    new FutureOutcome(outcomeF)
+  }
+
+  behavior of "Neutrino Node"
+
+  //assuming both peers are in sync with each other
+  it must "be able to sync" in { nodeConnectedWithBitcoinds =>
```

Comment:
Ig this test is a bit redundant since creating the fixture for `NeutrinoNodeTest` does have an `awaitSync`.

Reply:
I think its still valuable because if you introduce a regression, you can see that this specific test case is failing around syncing and then can deduce that is probably what is causing the rest of the test suites to fail.

Essentially it allows you to target the root cause of the regression faster, IMO.